flixel/input/gamepad/FlxGamepadManager.hx

```diff
@@ -368,7 +368,7 @@ class FlxGamepadManager implements IFlxInputManager
 	@:allow(flixel.FlxG)
 	private function new() 
 	{
-		#if FLX_OPENFL_JOYSTICK_API
+		#if (FLX_OPENFL_JOYSTICK_API && !next)
```

Comment:
I think we might be able to replace all the `#if (!flash && !next)` conditions by that too, making things more readable?